{"name": "[Negative] Verify user can't login using non-exist username", "status": "broken", "statusDetails": {"message": "KeyError: 'base_url'", "trace": "E   KeyError: 'base_url'\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "start": 1740992112952, "stop": 1740992112952, "uuid": "e118be06-8b04-4142-8afa-049879c0432d", "historyId": "42f9bad0da6b668bf1f97ba7126a78c1", "testCaseId": "42f9bad0da6b668bf1f97ba7126a78c1", "fullName": "tests.test_login#test_login_incorrect_username", "labels": [{"name": "story", "value": "User will failed login to website if credentials are incorrect"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "LOQ15"}, {"name": "thread", "value": "27276-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}